packages/block-library/src/paragraph/style.scss

```diff
@@ -28,6 +28,10 @@
 	font-style: normal;
 }
 
+:root p.has-line-height {
```

Comment:
Why is the `:root` necessary?

Reply:
It increases the specificity of the style for more consistent rendering in the editor and on the front-end (considering typical theme styles)

Reply:
Alright, then. Perhaps a comment may be helpful?